app/templates/suppliers/view_signed_agreement.html

```diff
@@ -4,21 +4,22 @@
   Countersign {{ framework.name }} agreement for {{ supplier_framework.declaration.nameOfOrganisation }} – Digital Marketplace admin
 {% endblock %}
 
-{% block breadcrumb %}
-    {%
-    with items = [
-        {
-            "link": url_for('.index'),
-            "label": "Admin home"
-        },
-        {
-            "link": url_for('.list_agreements', framework_slug=framework.slug),
-            "label": "%s agreements" | format(framework.name)
-        },
-        ]
-    %}
-        {% include "toolkit/breadcrumb.html" %}
-    {% endwith %}
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Admin home",
+        "href": url_for('.index')
+      },
+      {
+        "text": "%s agreements" | format(framework.name),
```

Comment:
I know this was pre-existing, but this seems like a good opportunity to be consistent in how we do our format syntax - in the file above we use "{} declaration".format(framework.name) which is more immediately recognisable for us Python folks 😸 Not a blocker however.